matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -539,9 +539,9 @@ var PhenoTips = (function (PhenoTips) {
         var phenScore = (Math.ceil(this._phenScoreSlider.value*20)/20).toFixed(2);
         var genScore = (Math.ceil(this._genScoreSlider.value*20)/20).toFixed(2);
 
-        var options = { 'score'        : score,
-                        'phenScore'    : phenScore,
-                        'genScore'     : genScore};
+        var options = { 'minScore'              : score,
+                        'minPhenotipicScore'    : phenScore,
+                        'minGenotipicScore'     : genScore};
```

Comment:
The param names were misspelled in the spec. Please rename:
* `minPhenotipicScore` -> `minPhenotypicScore` (or simply `minPhenScore`)
* `minGenotipicScore` -> `minGenotypicScore` (or simply `minGenScore`)